sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/implementation/converters/AttachmentTypeConverter.java

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.communication.chat.implementation.converters;
+
+import com.azure.communication.chat.models.AttachmentType;
+
+public final class AttachmentTypeConverter {
+    public static AttachmentType convert(com.azure.communication.chat.implementation.models.AttachmentType obj) {
+        if (obj == null) {
+            return null;
+        }
+
+        new AttachmentType();
+        AttachmentType attachmentType = AttachmentType.fromString(obj.toString());
+        return attachmentType;
```

Comment:
nit: you probably don't need `new AttachmentType()`   and can simplify it to
```suggestion
        return AttachmentType.fromString(obj.toString());
```